{
  try {
    setUpdateAvailable();
  }
 catch (  Exception e) {
    _log.error(e,e);
  }
}
